docs/getting-started-scalar-manager.md

```diff
@@ -0,0 +1,117 @@
+# Getting Started with Helm Charts (Scalar Manager)
+Scalar Manager is a web application providing a dashboard for users to check the health and paused status of Scalar products.
```

Comment:
```suggestion
Scalar Manager is a web-based dashboard that allows users to:
* check the health of the Scalar products
* pause and unpause the Scalar products to backup or restore underlying databases
* check the metrics and logs of the Scalar products through Grafana dashboards
```
BTW, I doubt Grafana explorers is proper naming.
Grafana dashboards?

Reply:
@feeblefakie 
I think the `Grafana dashboards` is more precise.
I used the `Grafana Explorer` because Scalar Manager uses the `Explore` subpage.